docs/examples/deploying_with_git _deploy.md

```diff
@@ -0,0 +1,127 @@
+# Deploying with Git Deploy
+
+This guide shows you how to deploy with [git-deploy](https://github.com/mislav/git-deploy). 
+
```

Comment:
Set up the high-level expectations early on in the article. This allows the reader to set up a proper mind frame for what is coming, and what will be the output when the article is followed.
 
```
This guide shows you how to deploy with [git-deploy](https://github.com/mislav/git-deploy). 

To set up git-deploy on Semaphore, we will set up:

1. Create a Git Deploy key that allows pushing to your production Git server. 
2. Store the Git Deploy key in a [Secret](secret) on Semaphore.
3. Create a deployment pipeline, and attach the Git Deploy key secret.
4. Run a deployment from Semaphore, and ship your code to production.

For this guide you will need:

- [A working Semaphore project][create-project] with a basic CI pipeline. 
You can use one of the documented [use cases][use-cases] or [language guides][language-guides] as a starting point.
- Basic familiarity with Git and SSH.
```